src/wp-includes/class-wp-theme-json.php

```diff
@@ -2128,9 +2144,33 @@ protected static function compute_style_properties( $styles, $settings = array()
 			}
 		}
 
+		if ( $can_use_cached ) {
+			set_site_transient( $cache_key, $declarations, HOUR_IN_SECONDS );
+		}
+
 		return $declarations;
 	}
 
+	/**
+	 * Gets the hash of the global settings.
+	 *
+	 * @since 6.6.0
+	 *
+	 * @return string The hash of the global settings.
+	 */
+	private function wp_get_global_settings_hash() {
```

Comment:
`get_global_settings` is a part of `theme_json` group. This function matches with the lifecycle of the hashed array. 
`md5` cost increases with an increase in the size of the string. Since we were computing md5 almost every time (85 times) for this large array, we had an unnecessary overhead. Caching the hash resulted in a `17%` improvement ( initially response median was 377, after this code it was 310 ).